pkg/lib/config/configsource/resources.go

```diff
@@ -12,24 +13,39 @@ import (
 	"github.com/authgear/authgear-server/pkg/api/apierrors"
 	"github.com/authgear/authgear-server/pkg/api/model"
 	"github.com/authgear/authgear-server/pkg/lib/config"
+	portalmodel "github.com/authgear/authgear-server/pkg/portal/model"
```

Comment:
pkg/lib cannot import pkg/portal. The workaround is to move Domain to pkg/api/model.

Reply:
Updated path and removed original workaround